hashmap.js

```diff
@@ -166,11 +166,14 @@
 		}
 	};
 
+	// ES6 Map API compatibility
+	HashMap.prototype.remove = HashMap.prototype.delete;
+
 	HashMap.uid = 0;
 
 	//- Add chaining to all methods that don't return something
 
-	['set','multi','copy','remove','clear','forEach'].forEach(function(method) {
+	['set','multi','copy','delete','remove','clear','forEach'].forEach(function(method) {

```

Comment:
Rather move `HashMap.prototype.remove = HashMap.prototype.delete;` below this block. We want both functions to be the same